frontend/public/components/search.tsx

```diff
@@ -200,6 +204,33 @@ const SearchPage_: React.FC<SearchProps & StateProps & DispatchProps> = (props)
     );
   };
 
+  const getCreateLinkButton = (kind: string) => {
+    const ko = findKindObj(kind) as K8sKind;
+    const { label, plural } = ko;
+    let href = namespace
+      ? `/k8s/ns/${namespace || 'default'}/${plural}/~new`
+      : `/k8s/cluster/${plural}/~new`;
+    if (ko.crd) {
+      try {
+        const ref = referenceForModel(ko);
+        href = namespace
+          ? `/k8s/ns/${namespace || 'default'}/${ref}/~new`
+          : `/k8s/cluster/${ref}/~new`;
+      } catch (unused) {
+        /**/
+      }
+    }
+    const createProps = { to: href };
+    const createLink = (
+      <RequireCreatePermission model={ko} namespace={namespace}>
+        <Link className="co-m-primary-action" {...createProps}>
```

Comment:
```suggestion
        <Link className="co-m-primary-action" to={href}>
```

Reply:
This isn't right for any resource that has a create dropdown like secrets.

Reply:
This will also show the create button for any resource that has intentionally disabled it like the ClusterServiceVersions resource.

Reply:
@spadgett - yeah, this solution of putting the `creat` button in the accordion toggle from the `search` page point of view is not going to work.  I think the options at this point are to either 
1) Put the accordion logic in the ListPage view or 
2) Make the existing `Create` button appear on the toogle through css magic, although I am not sure how to make that responsive.  
Any thoughts on a better solution?

Reply:
Yeah, this is why we've avoided it to this point. It seems like a small thing (just moving a button), but it's a significant code change since the logic for create is in the list component.

(1) seems workable. I wouldn't go down the road of (2).

I'm tempted to say that we shouldn't worry about the few special cases for now, but it would be a regression from 4.4 :/

The special cases that I'm aware of at the moment are

* Secrets
* Nodes
* ClusterServiceVersions
* Role Bindings
* Namespaces
* Projects
* ImageManifestVulns

There are probably others from plugins.

Reply:
I'd suggest fixing the filter row bug now and we move the create button in a follow on.

Reply:
Agreed on separating these out especially since there are 2 bugs included in this PR currently.  I will leave this open for the filter removal since that it already has that as the primary bug and remove the move create button code.